### PR TITLE
Add Formspree

### PR DESCRIPTION
**Pull request by clarmso**
Formspree's free tier allows us to send email using an HTTP request.  It's useful for front-end devs who needs to send emails from a web form.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -606,6 +606,7 @@ Table of Contents
   * [readme.io](https://readme.io/) — Beautiful documentations made easy, free for Open Source
   * [formaholic.com](https://formaholic.com) — Simple form endpoint. Perfect for static sites
   * [http2.pro](https://http2.pro) — HTTP/2 protocol readiness test and client HTTP/2 support detection API.
+  * [Formspree.io](https://formspree.io/) — Send email using an HTTP POST request.
```
Comment (ripienaar) [inline README.md]: also be nice to know about the limits of the free plan - 1000 a month

**Comment by clarmso** [comment]
Added description for the free plan/tier.

**Comment by ripienaar** [comment]
unfortunately this one needs a rebase against master other changes invalidated it :(

**Comment by clarmso** [comment]
Done! :)

**Comment by ripienaar** [comment]
thank you

**Pull request merged**